### ISSUE TITLE
Command always detects one table

### ISSUE THREAD

**Issue by MahmoudKhaledAli** [o1lab/xmysql#40]
node -v
( xmysql requires node >= 7.6.0 )
v8.11.1

npm -v
5.6.0
mysql --version
8.0.11
xmysql --version
0.4.8

- - -

When I run the command given in the README, the tool detecs one table only and generates 29 APIs, all of which are referencing a table named 'undefined', however the /tables API returns the correct database tables but their APIs are not generated.
I tried on a database of mine as well as classicmodels database that is used in the tests.

- - -

**Comment by o1lab** [comment]
@MahmoudKhaledAli : are there any warnings after command is started ?

also please paste the command you are using

**Comment by MahmoudKhaledAli** [comment]
I got no warning, just that the number of tables is 1 and it generated 29 APIs.
This is the command:
````
xmysql -u root -p root -d classicmodels
````

**Comment by o1lab** [comment]
Can you try /api/orders/describe - if it fails - then there is issue connecting to example with credentials. Can you ensure mysql schema is the right ones. Try any mysql client and see if it works fine.

**Comment by MahmoudKhaledAli** [comment]
I tried and got cannot GET error.
I tried both MySQL Workbench and MySQL command line tools and the credentials (root, root) are working fine. I also tried from Node.js with this config:
````
const LocaldbConfig = {
    host: "localhost",
    user: "root",
    password: "root",
    database: "games",
    port: 3306,
    dateStrings: "date"
};
````
And it's working fine.

**Comment by biwineGitAdmin** [comment]
Are you emailing the correct people ?

**Comment by o1lab** [comment]
Got a clue that mysql 8 returning some of column value in caps - which is resulting a problem in error generation. I have provided a fix in version 0.4.9

`npm update -g xmysql`

Try and let me know.

**Comment by MahmoudKhaledAli** [comment]
It's working now. Thank you.
